tests/jballerina-unit-test/src/test/resources/test-src/expressions/stamp/json-stamp-expr-test.bal

```diff
@@ -109,6 +109,20 @@ function stampJSONArrayToConstraintArray() returns Student []|error{
     return studentArray;
 }
 
+function stampTupleToJSON() {
+    [string, string, string] tupleValue = ["Mohan", "single", "LK2014"];
+    json|error jsonValue = tupleValue.cloneWithType(json);
+
+    assert(jsonValue is error, false);
+
+    json tempVal = null;
+    if (jsonValue is json) {
+        tempVal = jsonValue;
+    }
```

Comment:
Can't we check for `json[]` and make `tempVal` be of type `json[]` instead.

You can get rid of the casts below then.